cassandra-protocol/src/frame/frame_result.rs

```diff
@@ -94,6 +94,33 @@ pub enum ResResultBody {
     SchemaChange(SchemaChange),
 }
 
+impl Serialize for ResResultBody {
+    #[inline]
+    fn serialize(&self, cursor: &mut Cursor<&mut Vec<u8>>) {
+        match &self {
+            ResResultBody::Void(_) => {
+                to_int(1).serialize(cursor);
```

Comment:
to_int allocates and should not be used.

Reply:
That's right - primitive types already impl Serialize, so you can use it directly. In fact, I wish to take a look at those `to_*` functions and see if they are needed anyway.